packages/big-design/src/components/List/GroupHeader/ListGroupHeader.tsx

```diff
@@ -5,7 +5,7 @@ import { StyledGroupHeader } from './styled';
 export const ListGroupHeader: React.FC<React.LiHTMLAttributes<
   HTMLLIElement
 >> = memo(({ className, style, value, ...rest }) => (
-  <StyledGroupHeader {...rest} tabIndex={-1} onMouseDown={preventFocus} role="group" />
+  <StyledGroupHeader {...rest} tabIndex={-1} onMouseDown={preventFocus} aria-hidden />
```

Comment:
`aria-hidden` actually needs a value, otherwise the user agent determines if the element is exposed to or hidden from the accessibility API.
```suggestion
  <StyledGroupHeader {...rest} tabIndex={-1} onMouseDown={preventFocus} aria-hidden={true} />
```